eng/common/templates/variables/docker-images.yml

```diff
@@ -1,5 +1,5 @@
 variables:
-  imageNames.imageBuilderName: mcr.microsoft.com/dotnet-buildtools/image-builder:2508988
+  imageNames.imageBuilderName: mcr.microsoft.com/dotnet-buildtools/image-builder:2511639
```

Comment:
This PR needs the changes from https://github.com/dotnet/docker-tools/pull/1382 to flow in first.